nvbios/info.c

```diff
@@ -249,6 +249,11 @@ int envy_bios_parse_bit_i (struct envy_bios *bios, struct envy_bios_bit_entry *b
 			bios->chipset = 0xf0;
 			bios->chipset_name = "GK110";
 			break;
+		/* GK208 */
+		case 0x8028:
```

Comment:
Please put this in chip order. As another example of this, see GF100 above which is placed after the teslas. Otherwise it gets confusing. i.e. this should go after the GK110B entry.